app/src/main/cpp/skyline/services/am/controller/ILibraryAppletCreator.cpp

```diff
@@ -6,21 +6,23 @@
 #include "ILibraryAppletCreator.h"
 
 namespace skyline::service::am {
-    ILibraryAppletCreator::ILibraryAppletCreator(const DeviceState &state, ServiceManager &manager) : BaseService(state, manager, Service::am_ILibraryAppletCreator, "am:ILibraryAppletCreator", {
+    ILibraryAppletCreator::ILibraryAppletCreator(const DeviceState &state, ServiceManager &manager) : BaseService(state, manager, {
         {0x0, SFUNC(ILibraryAppletCreator::CreateLibraryApplet)},
         {0xA, SFUNC(ILibraryAppletCreator::CreateStorage)}
     }) {}
 
-    void ILibraryAppletCreator::CreateLibraryApplet(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+    Result ILibraryAppletCreator::CreateLibraryApplet(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
         manager.RegisterService(SRVREG(ILibraryAppletAccessor), session, response);
+        return {};
     }
 
-    void ILibraryAppletCreator::CreateStorage(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+    Result ILibraryAppletCreator::CreateStorage(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
         auto size = request.Pop<i64>();
 
         if (size < 0)
             throw exception("Cannot create an IStorage with a negative size");
```

Comment:
Is there no appropriate error for this?

Reply:
This code isn't even part of the pr ;)

Reply:
I'm aware but this is a refactor so mostly anything goes